util/chef-install.sh

```diff
@@ -630,48 +660,46 @@ fi
 # $tmp_dir:
 #
 # Outputs:
-# $download_url:
-# $sha256:
+# $download_url: Url of the package to download
+# $sha256: SHA256 checksum of the file to download
 ############
+bucket_url="https://${region}-aws-parallelcluster.${region}.${download_domain}/packages/${project}/"
```

Comment:
missing `.s3` in the URL, correct syntax is
`https://<bucketName>.s3.<region>.amazonaws.com/<keyName>`